demo-swarm/start-demo-swarm.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+echo "----- Trying to init swarm... (will fail if already part of one)"
+sudo docker swarm init
+
+#echo "----- Creating registry for images"
```

Comment:
I'm leaving the registry creating scripts as comments since we will probably use that in the future